angular-site/src/app/data-table/paginator/paginator-configurable-example.ts

```diff
@@ -0,0 +1,26 @@
+import {Component} from '@angular/core';
+import {PageEvent} from '@angular/material/paginator';
+
+/**
+ * @title Configurable paginator
+ */
+@Component({
+  selector: 'paginator-configurable-example',
+  templateUrl: 'paginator-configurable-example.html',
+  styleUrls: ['paginator-configurable-example.css'],
+})
+export class PaginatorConfigurableExample {
```

Comment:
add new line before

Reply:
According to the [Angular coding style guide](https://angular.io/guide/styleguide) new line is not needed here. Do you think I should add it nevertheless?